src/mrinufft/trajectories/utils.py

```diff
@@ -1,5 +1,6 @@
 """Utility functions for the trajectory design."""
 
+import numbers as nb
```

Comment:
`nb` is often used for numba. I would rather import the numbers type you want directly. 

```suggestion
from numbers import Real
```